### ISSUE TITLE
Version 1.3.1 of package fails to build

### ISSUE THREAD

**Issue by Venryx** [node-3d/segfault-raub#9]
Yarn: v3.3.0
OS: Windows 10

I have segfault-raub as a dependency of my project. When I pin it to version 1.2.0, it builds fine during `yarn install`; when I update it to v1.3.1, I get this error:
```
➤ YN0007: │ segfault-raub@npm:1.3.1 must be built because it never has been before or the last one failed
➤ YN0000: │ segfault-raub@npm:1.3.1 STDERR To build addons, `node-addon-api` module is required.
➤ YN0009: │ segfault-raub@npm:1.3.1 couldn't be built successfully (exit code 1, logs can be found here: C:\Users\Venryx\AppData\Local\Temp\xfs-66259cf4\build.log)  
```
(and the log file mentioned just repeats the error above, of: `To build addons, `node-addon-api` module is required.`)

EDIT: Also tried with v1.3.0, and same error as for 1.3.1.

For now, I am simply sticking with v1.2.0 in my projects, but I figured I would mention the issue.

**Comment by raub** [comment]
Hello. As a workaround, you may also add module `node-addon-api` to your list of DEV dependencies. I also came across this issue and found that this happens in a deeper dependency. It should be fairly easy to resolve though, so I will follow up when done.

**Comment by raub** [comment]
The error should have been fixed by `segfault-raub@1.3.2`
